docs/using-est-with-certd.md

```diff
@@ -0,0 +1,187 @@
+# Using EST with Certificates Service
+
+This document covers how to configure Certificates Service to issue certificates over EST.
+
+## Prerequisites
+
+- Certificates Service and Keys Service configured and running
+- A working EST server that processes enrollment requests
+
+## Configuration Options
+
+EST certificate issuance is configured in Certificates Service's config.toml, which by default is `/etc/aziot/certd/config.toml` or a file in `/etc/aziot/certd/config.d/`.
```

Comment:
Just to confirm... the config files we're telling a customer to modify here would not be overwritten by someone applying their uber config, right?

Reply:
The uber config only modifies `/etc/aziot/certd/config.d/00-super.toml`, so changes won't be overwritten as long as the customer chooses another file name.